pkg/customresourcestate/registry_factory.go

```diff
@@ -90,10 +90,13 @@ func compileFamily(f Generator, resource Resource) (*compiledFamily, error) {
 	if errorLogV == 0 {
 		errorLogV = resource.ErrorLogV
 	}
+
+	help := fmt.Sprintf("%s for %s/%s/%s", f.Help, resource.GroupVersionKind.Group, resource.GroupVersionKind.Version, resource.GroupVersionKind.Kind)
```

Comment:
Lets stick to the standardized representation here: https://github.com/kubernetes/apimachinery/blob/v0.33.0/pkg/apis/meta/v1/group_version.go#L93.

Reply:
done